packages/subgraph/manifest/subgraph.placeholder.yaml

```diff
@@ -68,6 +68,14 @@ templates:
           handler: handleRevoked
         - event: Frozen(indexed bytes32,indexed address,address)
           handler: handleFrozen
+        - event: Withdrawn(indexed address,indexed address,uint256,string)
```

Comment:
We didn't have withdraw before, as it is not a new event?
I think we decided to handle withdraw differently via `if (methodSig == '0x4f065632')` on `handleExecuted`
because there was some data we could not aggregate properly otherwise.

please compare.

Reply:
you are right, but this wouldn't catch withdrawns that happen with the Withdrawn Permission.
I adjusted the handling of withdrawls a bit. can you check again?
if that is fine for you I'll adjust the tests

Reply:
true, ok perfect now